app/src/hooks/useStakingOptions.ts

```diff
@@ -41,23 +47,35 @@ async function fetchData(provider: AnchorProvider): Promise<SoParams[]> {
   ];
   const prices = await fetchMultiBirdeyePrice(quoteMints);
 
+  const soMints = await Promise.all(
+    states.flatMap((state) =>
+      // @ts-ignore
+      state.strikes.map((strike) => stakingOptionsHelper.soMint(strike, state.soName, state.baseMint))
+    )
+  );
+
+  const soMintAccounts = (await provider.connection.getMultipleParsedAccounts(soMints)).value.reduce<{
```

Comment:
you need to chunk this. different rpc providers have different limits. 100 for most, but quiknode recently switched to 5